src/js/components/projects/detail.tsx

```diff
@@ -80,6 +89,21 @@ const ProjectDetail = (props: RouteComponentProps) => {
   const openCreateModal = () => setCreateModalOpen(true);
   const closeCreateModal = () => setCreateModalOpen(false);
 
+  const closeTourLandingModal = () => setTourLandingModalOpen(false);
+  const doRunTour = (type: string) => {
+    switch (type) {
+      case 'plan':
+        closeTourLandingModal();
+        setRunTour(true);
+    }
+  };
+  const handleTourCallback = (data: CallBackProps) => {
+    const { status } = data;
+    const finishedStatuses: string[] = [STATUS.FINISHED, STATUS.SKIPPED];
+    if (finishedStatuses.includes(status)) {
+      setRunTour(false);
```

Comment:
@jgerigmeyer this is probably the point in which we should 'PUT' to the user endpoint to update" `onboarding_at` ? I'm wondering how we get someone to do onboarding for 'Play' and 'Help'

Reply:
> @jgerigmeyer this is probably the point in which we should 'PUT' to the user endpoint to update" `onboarding_at` ?

@duggiemitchell Actually I think we want to set this initially when the modal opens. Even if a user doesn't complete the walkthrough, we don't want to show them this modal a second time.

> I'm wondering how we get someone to do onboarding for 'Play' and 'Help'

The little `?` dropdown menu needs to be added to the global header, next to the user icon. Users can't get back to the landing modal, but they can directly start any of the walkthroughs via that dropdown.